src/layouts/components/Header.tsx

```diff
@@ -225,7 +262,7 @@ function TabletMenu({ toggleMenu }: { toggleMenu: () => void }): JSX.Element {
           <MdClose
             role="button"
             className="h-6 w-6 text-primary-500"
-            onClick={() => toggleMenu()}
+            onClick={() => props.toggleMenu}
```

Comment:
```suggestion
            onClick={() => props.toggleMenu()}
```

toggleMenu is a void function so would need the () for the close button to work for the tablet menu

Reply:
done!